include/arkode/arkode_lsrkstep.h

```diff
@@ -0,0 +1,91 @@
+/* -----------------------------------------------------------------
+ * Programmer(s): Daniel R. Reynolds and Mustafa Aggul @ SMU
+ * -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2024, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------
+ * This is the header file for the ARKODE LSRKStep module.
+ * -----------------------------------------------------------------*/
+
+#ifndef _LSRKSTEP_H
+#define _LSRKSTEP_H
+
+#include <arkode/arkode.h>
+
+#ifdef __cplusplus /* wrapper to enable C++ usage */
+extern "C" {
+#endif
+
+typedef int (*ARKDomEigFn)(sunrealtype t, N_Vector y, N_Vector fn,
+                           sunrealtype* lambdaR, sunrealtype* lambdaI,
+                           void* user_data, N_Vector temp1, N_Vector temp2,
+                           N_Vector temp3);
+
+/* ------------------
+ * LSRKStep Constants
+ * ------------------ */
+
+typedef enum
+{
+  ARKODE_LSRK_RKC_2    = 1, /* ensure enum is int */
```

Comment:
Your comment is that you want to 'ensure enum is int". Firstly, why do you need to guarantee the type for the enum?  Secondly, do you mean that you want to ensure that it is a signed integer type? If so, the code is not sufficient to do that. To ensure that the type is signed, you will need to include some enum value that is negative.